docs/common_functions.md

```diff
@@ -0,0 +1,252 @@
+---
+id: common_functions
+title: 5 - Common Functions in Chialisp
+---
+
+When you start to write full smart contracts, you will start to realize that you will need certain common functionality in a lot of puzzles.  Let's go over how to include them and what some of them are:
+
+## include
+
+If you want to import some functionality that you use frequently without having to copy/paste it between files, you can use `include`:
+
+```lisp
+;; condition_codes.clvm
+(
+  (defconstant AGG_SIG_ME 50)
+  (defconstant CREATE_COIN 51)
+)
+```
+
+```lisp
+;;main.clvm
+(mod create_sig_coin (pubkey msg puzzle_hash amount)
+
+  (include "condition_codes.clvm")
+
+  (list (list AGG_SIG_ME pubkey msg) (list CREATE_COIN puzzle_hash amount))
+
+)
+```
+
+When running main.clvm with `run`, make sure to use the `-i` option to specify in which directories to look for files.
+
+Also note that the include files are a special format. All functions and constants defined go in a set of parentheses.
```

Comment:
Maybe clarify this line? All functions and constants (and everything else lol) go inside parentheses. Why/how are include files special format?